Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml

```diff
@@ -13,6 +13,45 @@
     - sprite: Mobs/Silicon/chassis.rsi
       state: medical_e
 
+- type: entity
+  id: SpawnPointChiefJustice
+  parent: SpawnPointJobBase
+  name: chief justice
```

Comment:
```suggestion
  name: chiefjustice
```
![image](https://github.com/DeltaV-Station/Delta-v/assets/113523727/827ea9c9-ea2a-4418-afc6-0118392fdbbe)

to keep it consistent

Reply:
Resolved, on top of the other locker comments